modules/docs/arrow-docs/docs/docs/typeclasses/traverse/README.md

```diff
@@ -9,8 +9,259 @@ permalink: /docs/typeclasses/traverse/
+import arrow.instances.either.applicativeError.applicativeError
+import java.lang.NumberFormatException
+import arrow.instances.list.applicative.*
+import arrow.instances.list.foldable.sequence_
+import arrow.instances.list.foldable.traverse_
+import arrow.instances.list.traverse.sequence
+import arrow.instances.list.traverse.traverse
+import arrow.instances.validated.applicative.applicative
+import arrow.instances.nonemptylist.semigroup.semigroup
+import arrow.instances.option.applicative.applicative
+import kotlinx.coroutines.*
+```
+
+Next, we will define some data classes and functions to show case different examples of traverse below.
+
+```kotlin:ank:silent
+sealed class SecurityError {
+    data class RuntimeSecurityError(val cause: String) : SecurityError()
+}
+
+interface Credential
+data class Profile(val id: String)
+data class User(val id: String, val name: String)
+```
+
+`parseInt`: is a function that will try to convert an `String` parameter `s` to an `Int`, it if succeeds it will return the number inside the `Some`. If it fails, it will return `None`.
+
+ `validateLogin`: This function can be something that could either fail or be a sauccessful operation when validating login credentials */
+
+`userInfo`: This function could be something that could return a Profile asynchronously.
+
+```kotlin:ank:silent
+interface OurSideEffectsFunctions {
+    fun parseInt(s: String): Option<Int> =
+            Try { s.toInt() }.fold(ifFailure = { None }, ifSuccess = { v -> Some(v) })
+
+    fun validateLogin(cred: Credential): Either<SecurityError, Unit>
+
+    fun userInfo(user: User): DeferredK<Profile>
+}
+```
+
+As we can see, every function above only takes as parameter the argument to perform its operation on.
+
+The next objects are just for mocking and simulate results of a side effect performed by an external system so we can see the output of our examples.
+
+```kotlin:ank:silent
+object ValidEffects : OurSideEffectsFunctions {
+
+    override fun validateLogin(cred: Credential): Either<SecurityError, Unit> {
+        return Either.right(Unit)
+    }
+
+    override fun userInfo(user: User): DeferredK<Profile> {
+        return DeferredK.async { Profile(id = user.id) }
+    }
+
+    fun savingProfiles(): DeferredK<Unit>  = GlobalScope.async(Dispatchers.Default, CoroutineStart.LAZY){Unit}.k()
+
+}
+```
+
+We just defined results for our functions when everything went "right", the next object is also just for simulating the result when something goes "wrong".
+
+```kotlin:ank:silent
+object ErrorEffects : OurSideEffectsFunctions {
+
+    override fun validateLogin(cred: Credential): Either<SecurityError, Unit> {
+        return Either.left(SecurityError.RuntimeSecurityError("Invalid credentials"))
+    }
+
+    override fun userInfo(user: User): DeferredK<Profile> {
+        return DeferredK.failed(Throwable("Error retrieving profile"))
+    }
+}
+```
+
+If you want to run these examples you can run them inside a main body.
```

Comment:
I'm not sure what's the intention here, but if you wanna have a runnable version why don't you provide the code? (you're leaving it empty). Not sure whether we should just remove this instead. What's the playgrounds support status @raulraja ?

Reply:
I will remove this.